### DISCUSSION TITLE
Suppport for Hihome Outdoor Wifi AppCam Smart HD

### DISCUSSION THREAD

**Discussion by tomgit33** [roleoroleo/yi-hack-Allwinner-v2#805] [Ideas]
Support for this Cam would be awesome:
[https://www.ebay.de/itm/225809464923](https://www.ebay.de/itm/225809464923)
[https://en.hihome.eu/products/hihome-outdoor-camera-hd](https://en.hihome.eu/products/hihome-outdoor-camera-hd)

Specs seeming similar to yi 1080p outdoor.  :)

Thank you

**Comment by roleoroleo** [comment]
Try to use the backup procedure described in the wiki.

**Reply by tomgit33** [reply]
OK, i will buy one and upload the firmware. Maybe next Weekend. 
Thank you

**Reply by roleoroleo** [reply]
Or, if you can, open it and take some pictures.

**Comment by tomgit33** [comment]
Backup:
I think the Backup didn't work
[HIhome wsc-od1.zip](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/14088210/HIhome.wsc-od1.zip)
![HIhome wsc-od1](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/155909174/c809ac00-54ba-4978-9b98-693e53e308eb)

Can i do anything else?

**Reply by roleoroleo** [reply]
Backup didn't work.
Probably it's a different hardware platform.
